canvas_modules/harness/test_resources/parameterDefs/passwordfield_paramDef.json

```diff
@@ -442,5 +442,9 @@
         }
       }
     }
-  ]
+  ],
+  "resources": {
+    "pwd.passwordShow.tooltip": "Show password",
```

Comment:
Please change this value to someething else. Because "Show password" is a default value.